go.mod

```diff
@@ -166,6 +166,8 @@ replace (
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
 	github.com/jhump/protoreflect => github.com/jhump/protoreflect v1.9.0
+	// use informal system fork of tendermint
+	github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.24
```

Comment:
Are all chains that upgrade to v0.47 doing to have to do this as well? If so, we should reference this in the release notes & upgrading guide (if we have not already).

Reply:
That is a good one, let me add that in the release notes. We should do that for v0.45 too.

Reply:
do we need this for 0.46 and 0.45? I would think not?

Reply:
The replace or the mention?

Reply:
the replace, we can mention it, but its also not required

Reply:
Any where we have a `replace` I think a chain would need a replace too because Tendermint dep is used in app-specific stuff like cmd. So updating our docs/guides will be necessary.

Reply:
I think the advice is to use the replace given that you could rewrite git history (go.sum should detect that but still - for new chains you don't have that garantee).